_posts/2019-12-13-basic-rails-container.markdown

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: "A Basic Rails Container"
+date: 2019-12-13 12:30:00
+categories: ["DevOps", "docker", "tutorials"]
```

Comment:
@rdormer Can you make `DevOps` lowercase here and in all previous articles?
I just found out that that tag is failing because of that (https://www.ombulabs.com/blog/tags/DevOps)